common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -573,6 +573,12 @@
     },
     "sqlState" : "58030"
   },
+  "CANNOT_WRITE_VIEW" : {
+    "message" : [
+      "Writing into a view is not allowed. View: <identifier>."
+    ],
+    "sqlState" : "0A000"
```

Comment:
This error class `0A` is about unsupported feature. Do you this it could be potentially supported in the future? If not, could you consider another error class, for instance `42`